extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -125,6 +125,13 @@ export class AmpSelector extends AMP.BaseElement {
       const delta = (args && args['delta'] !== undefined) ? args['delta'] : 1;
       this.select_(delta);
     }, ActionTrust.LOW);
+
+    this.registerAction('toggle', invocation => {
+      const args = invocation.args;
+      if (args && args['index'] !== undefined) {
+        this.toggle_(args['index'], args['value']);
```

Comment:
Let's do a user assert to make sure the `index` is valid

Reply:
Added an assert to make sure that the `index` is > 0, could also add one to make sure it is less than length of the `options`

Reply:
I think that both bounds would be helpful. If it ends up being too noisy we can downgrade it to a `user().warn()` later.